container/mutagen.yml

```diff
@@ -30,10 +30,11 @@ actions:
       location: /keg/tap
       privileged: true
       cmds:
-        - yarn roll:dev
+        - yarn build
     start:
       location: /keg/tap
       privileged: true
```

Comment:
Not super important, but `privileged` is now on by default, and you have to explicitly turn it off. So you can remove the `privileged: true` if you want

Reply:
ok. updated